libqtile/core/manager.py

```diff
@@ -1138,14 +1139,24 @@ def cmd_spawn(self, cmd, shell=False):
 
             spawn(["xterm", "-T", "Temporary terminal"])
         """
+        to_lookup = ""
         if shell:
             if not isinstance(cmd, str):
+                to_lookup = cmd[0]
                 cmd = subprocess.list2cmdline(cmd)
+            else:
+                to_lookup = cmd
```

Comment:
```suggestion
                to_lookup = cmd.split(" ", 1)[0]
```

Reply:
Really it should be shlex.split(cmd)[0], right?

Reply:
yeah preferably I guess, although it probably doesn't change anything because we're looking for the binary name

Reply:
It does if the binary name has a space in it :)